### ISSUE TITLE
[Question] ignore bookmark source / content

### ISSUE THREAD

**Issue by IDDQD69** [xwmx/nb#249]
Hi, 

Is there a way to not fetch url source / content, but still fetch title etc?

**Comment by maxdiebold** [comment]
I'm not sure if my request is the same, but I'd really appreciate an option to only save the title, URL, and tags of bookmarks, since I very rarely search by or view the content.  I keep a separate notebook just for bookmarks, and search by title/tags for opening the links in a browser.

I haven't looked into how the tool performs the searches when the `-t` flag is passed, but I imagine search speed could be increased by trimming off the content?

**Comment by j-krl** [comment]
Old ticket but I'm interested in this as well. @xwmx would you accept a simple PR that added a `--no-content` flag to `nb bookmark`?

**Comment by xwmx** [comment]
There actually is an undocumented option for this. It should work with either `--no-content` or `--skip-content`. Let me know if it doesn’t work. I’ll look into adding it to the documentation and whether tests are still needed for it.

**Comment by j-krl** [comment]
Yes, both those flags work perfect. Issue can probably be closed
